### DISCUSSION TITLE
R 5.3.0 Sunbird Inquiry QA Regression Status

### DISCUSSION THREAD

**Discussion by Nadiya496** [Sunbird-inQuiry/Community#48] [General Discussions]
Hello All,

Please find the testing status of R 5.3.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (12.90%)
Planned Rgression Completion-Off Date: December 26, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10616)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pt4jdEWP-9Cp-w1eJoa9zcSHsUZsSpM4SDPv9tLpMGg/edit#gid=0)

Execution Details:
![image](https://user-images.githubusercontent.com/53341543/207632740-724b0ac6-5759-48bc-8ef2-571e7db9216f.png)

cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav, @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @Nadiya496 @PoojaJManjooran

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.3.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (23.99%)
Planned Rgression Completion-Off Date: December 26, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10616)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pt4jdEWP-9Cp-w1eJoa9zcSHsUZsSpM4SDPv9tLpMGg/edit#gid=0)

Execution Details:

![image](https://user-images.githubusercontent.com/53341543/207865172-d5d2bc4d-0904-4d3e-984f-f071f9759d1e.png)


cc: @pallakartheekreddy @Aparna-karri591 @anunagaraj @ShailandraOjha @chitranshu-keshav, @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @Nadiya496 @PoojaJManjooran

**Comment by Aparna-karri591** [comment]
Hello All,

Please find the testing status of R 5.3.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (34.88%)
Planned Regression Completion-Off Date: December 26, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10616)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pt4jdEWP-9Cp-w1eJoa9zcSHsUZsSpM4SDPv9tLpMGg/edit#gid=0)

Execution Details:

![image](https://user-images.githubusercontent.com/106593060/208101135-34814515-aa91-46d0-9f5f-dc290d3a1fa4.png)

cc: @pallakartheekreddy @anunagaraj @ShailandraOjha @chitranshu-keshav, @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @Nadiya496 @PoojaJManjooran

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.3.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (44.96%)
Planned Regression Completion-Off Date: December 26, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10616)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pt4jdEWP-9Cp-w1eJoa9zcSHsUZsSpM4SDPv9tLpMGg/edit#gid=0)

Execution Details:
![image](https://user-images.githubusercontent.com/53341543/208468097-3b8686f2-f82e-4df5-8cdd-1cf72e677fc8.png)
cc: @pallakartheekreddy @anunagaraj @ShailandraOjha @chitranshu-keshav, @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @Nadiya496 @PoojaJManjooran

**Comment by Aparna-karri591** [comment]
Hello All,

Please find the testing status of R 5.3.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (57.26%)
Planned Regression Completion-Off Date: December 26, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10616)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pt4jdEWP-9Cp-w1eJoa9zcSHsUZsSpM4SDPv9tLpMGg/edit#gid=0)

Execution Details:

![image](https://user-images.githubusercontent.com/106593060/208680479-b583dc08-050c-4f48-b285-fdab58b204bc.png)

cc: @pallakartheekreddy @anunagaraj @ShailandraOjha @chitranshu-keshav, @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @Nadiya496 @PoojaJManjooran @Nadiya496

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.3.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (67.80%)
Planned Regression Completion-Off Date: December 26, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10616)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pt4jdEWP-9Cp-w1eJoa9zcSHsUZsSpM4SDPv9tLpMGg/edit#gid=0)

Execution Details:
![image](https://user-images.githubusercontent.com/53341543/208927439-129446cc-48c6-4bf8-a2f5-7586e1c5448d.png)

Defect Dashboard:
![image](https://user-images.githubusercontent.com/53341543/208927610-01609d1c-c2ef-4687-800a-861eb3434358.png)
cc: @pallakartheekreddy @anunagaraj @ShailandraOjha @chitranshu-keshav, @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @Nadiya496 @PoojaJManjooran @Aparna-karri591

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.3.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (80.77%)
Planned Regression Completion-Off Date: December 26, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10616)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pt4jdEWP-9Cp-w1eJoa9zcSHsUZsSpM4SDPv9tLpMGg/edit#gid=0)

**Execution Details:**
![image](https://user-images.githubusercontent.com/53341543/209145478-fa5921dd-a10b-45f8-a4c6-b1c68c1f3967.png)

**Defect Dashboard:**
![image](https://user-images.githubusercontent.com/53341543/209145632-fdc08b14-80f0-4cf5-af66-c5443a62ccb2.png)

cc: @pallakartheekreddy @anunagaraj @ShailandraOjha @chitranshu-keshav, @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @Nadiya496 @PoojaJManjooran @Aparna-karri591

**Comment by Nadiya496** [comment]
Hello All,

Please find the testing status of R 5.3.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (90.60%)
Planned Regression Completion-Off Date: December 26, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10616)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pt4jdEWP-9Cp-w1eJoa9zcSHsUZsSpM4SDPv9tLpMGg/edit#gid=0)

Execution Details:
![image](https://user-images.githubusercontent.com/53341543/209338617-061e142c-b800-4606-b010-be7a6fe010f7.png)
**Defect Dashboard:**
![image](https://user-images.githubusercontent.com/53341543/209145632-fdc08b14-80f0-4cf5-af66-c5443a62ccb2.png)
cc: @pallakartheekreddy @anunagaraj @ShailandraOjha @chitranshu-keshav, @rajnishdargan @surendrasinghs @sharathkashyap @itsvick @Ekta29yadav @Nadiya496 @PoojaJManjooran @Aparna-karri591
